client-side/C#/StreetSmartArcGISPro/StreetSmartArcGISPro/AddIns/DockPanes/StreetSmart.cs

```diff
@@ -1084,17 +1089,19 @@ private async void ViewerAdded(object sender, IEventArgs<IViewer> args)
         Setting settings = ProjectList.Instance.GetSettings(_mapView);
         Api.SetOverlayDrawDistance(settings.OverlayDrawDistance);
 
-        EventLog.Write(EventLog.EventType.Information, $"Street Smart: (StreetSmart.cs) (ViewerAdded) function get recording from panorama");
         IRecording recording = await panoramaViewer.GetRecording();
         string imageId = recording.Id;
+
         _viewerList.Add(panoramaViewer, imageId);
 
-        // ToDo: set culture: date and time
         Viewer viewer = _viewerList.GetViewer(panoramaViewer);
+
         ICoordinate coordinate = recording.XYZ;
         IOrientation orientation = await panoramaViewer.GetOrientation();
         Color color = await panoramaViewer.GetViewerColor();
+
         EventLog.Write(EventLog.EventType.Information, $"Street Smart: (StreetSmart.cs) (ViewerAdded) set coordinate, orientation and color");
+
         await viewer.SetAsync(coordinate, orientation, color, _mapView);
```

Comment:
Can we set a fake orientation here, instead of calling GetOrientation two times?